crates/rspack_plugin_copy/src/lib.rs

```diff
@@ -264,7 +264,12 @@ impl CopyRspackPlugin {
     logger.debug(format!("reading '{}'...", absolute_filename));
     // TODO inputFileSystem
 
-    let source_vec = match tokio::fs::read(absolute_filename.clone()).await {
+    #[cfg(not(target_family = "wasm"))]
```

Comment:
we should use compiler.fs in this plugin, but i will change this in following pr